ts-embed_assets/README.md

```diff
@@ -0,0 +1,61 @@
+# Embedding Assets Sample App
+
+## Run the app locally
+```sh
+ts-node index.ts
+```
+
+Test APIs with:
+```
+$ curl localhost:3000/hello
+hi%
+
+$ curl localhost:3000/static/hello.txt
+Hello Klotho!
+```
+
+## Compile and Deploy with Klotho
+
+run the terminal commands:
+```sh
+# Compile the app
+tsc && klotho . --app $USER-ts-embed-assets -p aws
+
+# Go into the compiled directory
+cd compiled
+
+# If you didn't set the aws region as indicated in the compiler output, do that now
+pulumi config set aws:region YOUR_REGION -s $USER-ts-embed-assets
+
+# npm install pulumi dependencies
+npm install
+
+# Deploy
+pulumi up -s $USER-ts-embed-assets
+
+# Outputs: {
+#   apiUrl: 'https://<...>.execute-api.us-east-1.amazonaws.com/stage/'
```

Comment:
Not sure if it matters, but all of the URLs here and below use `us-east-1`. I'm just calling that out because we explicitly tell them how to select their own region above, and then here we assume us-east-1.

I think it's fine?